app/repos/exemption.js

```diff
@@ -0,0 +1,67 @@
+const sequelize = require('../config/db')
+const { getCdo } = require('./cdo')
+const { getCourt, getPoliceForce } = require('../lookups')
+const { createInsurance, updateInsurance } = require('./insurance')
+
+const updateExemption = async (data, transaction) => {
+  if (!transaction) {
+    return sequelize.transaction((t) => updateExemption(data, t))
+  }
+
+  try {
+    const cdo = await getCdo(data.indexNumber)
+
+    if (!cdo) {
```

Comment:
Not a big issue but shouldn't we be returning a 400 rather than throwing an exception which will result in a 500? However, this looks clean so happy to leave as is